src/manager.rs

```diff
@@ -37,6 +37,16 @@ use log::{info, warn};
 #[cfg(test)]
 use std::{println as info, println as warn};
 
+macro_rules! info_with_replica {
+    ($replica_id:expr, $($arg:tt)*) => {
+        info!(
+            "[Replica {}] {}",
+            $replica_id,
```

Comment:
Can we make this macro access `self.replica_id` instead of taking it as a parameter?

Reply:
We would have to move `compute_quorum_results()` (https://github.com/pytorch/torchft/blob/082753cbe9705d00f63752e447aacfd6c21ca502/src/manager.rs#L357) into Manager class if we want to pass in the self instance, should we do that?

Reply:
ahh this is fine then -- was just curious if we could clean it up a bit but this is fine